### PR TITLE
updated lock directory for store package in Linux

### PR DESCRIPTION
**Pull request by tamilmani1989**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

**Comment by tamilmani1989** [comment]
pipeline triggered manually as it failing consistently when auto triggered: https://msazure.visualstudio.com/One/_build/results?buildId=44146602&view=results

File: platform/os_windows.go [hunk 1]
```diff
@@ -23,6 +23,9 @@ const (
 	// CNIRuntimePath is the path where CNI state files are stored.
 	CNIRuntimePath = ""
 
+	// CNILockPath is the path where CNI lock files are stored.
+	CNILockPath = ""
```
Comment (neaggarwMS) [inline platform/os_windows.go:27]: Should we make this also in separate dir?

Comment (tamilmani1989) [reply platform/os_windows.go:27]: this is windows and can update this when required. i leave it to current behaviour

**Review by neaggarwMS — APPROVED** [review]
:shipit:

**Pull request merged**